sympy/physics/continuum_mechanics/cross_section.py

```diff
@@ -0,0 +1,161 @@
+from sympy.geometry.polygon import Polygon, Point, Circle
+from sympy.core import sympify
+from sympy.printing import sstr
+from sympy.core import Symbol, S
+
+
+class CrossSection(object):
+    def __init__(self, shape, variable, *dimensions):
+        self.shape = shape
```

Comment:
I think this should be `self._shape = shape` because you have defined a property with the same name `shape`, so may be there will be conflict.